.github/workflows/ghc-lib-da-ghc-8.4.3.yml

```diff
@@ -11,53 +12,36 @@ jobs:
         os: [ubuntu, windows]
     steps:
       - uses: actions/checkout@v4
-      - name: Install msys2 (windows)
+      - name: Configure msys2 (windows)
         shell: bash
         run: |-
-          cd /c
```

Comment:
Why is it that we can get away with configuring msys instead of installing it here now?

Reply:
great question! see here https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md#msys2. it's because msys64 comes preinstalled on github runners but isn't added to the path.